schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  event:
```

Comment:
I think `event` is a misleading name for this meta data.  Can we call this `meta`?